### ISSUE TITLE
All new clients through the JSON api are password clients

### ISSUE THREAD

**Issue by daanh432** [laravel/passport#1245]
- Passport Version: 9.0.0
- Laravel Version: 7.10.2
- PHP Version: 7.2.11
- Database Driver & Version: Mysql 5.7.19

### Description:
Newly created OAuth clients are set to be password clients.

### Steps To Reproduce:
* [Install Laravel 7](https://laravel.com/docs/7.x/installation)
* Run ``composer require laravel/ui``
* Run ``php artisan ui vue --auth``
* Run ``composer require laravel/passport``
* Run ``php artisan migrate``
* Run ``php artisan passport:install``
* Add the Laravel\Passport\HasApiTokens trait to your App\User model
* Register the passport routes in the AuthServiceProvider
* Publish the frontend components ``php artisan vendor:publish --tag=passport-components``
* Add the following code block to your ``app.js``
```
Vue.component(
    'passport-clients',
    require('./components/passport/Clients.vue').default
);

Vue.component(
    'passport-authorized-clients',
    require('./components/passport/AuthorizedClients.vue').default
);

Vue.component(
    'passport-personal-access-tokens',
    require('./components/passport/PersonalAccessTokens.vue').default
);
```
* Add the following lines to your home.blade.php
```
<passport-clients></passport-clients>
<passport-authorized-clients></passport-authorized-clients>
<passport-personal-access-tokens></passport-personal-access-tokens>
```
* Run ``npm run dev``
* Register / login on your newly installed Laravel app
* Create a new OAuth client with the checkmark confidential checked.
* Look in your database and see that the new client is a password client.

### My Conclusion:

I think I've narrowed it down to the store function of the ``ClientController`` but I'm not 100% sure about it. To me, it looks like the new parameter for ``provider`` hasn't been properly implemented within the ``store`` method. The confidential boolean from the request gets entered into the password parameter.

#### Portion of code I'm talking about.
![image](https://user-images.githubusercontent.com/8281824/81197156-3beb7180-8fc0-11ea-83b8-eaa4b09372ef.png)

#### Database Example:
![image](https://user-images.githubusercontent.com/8281824/81197198-49a0f700-8fc0-11ea-8240-63d65388f257.png)

If there are any questions or uncertainties please let me know.

**Comment by driesvints** [comment]
Fixed in https://github.com/laravel/passport/commit/aff9d0933737354d04df98cfc431fa20309be03a

Thanks for reporting.

**Comment by driesvints** [comment]
Released v9.0.1 which fixes this.

**Comment by ERROUDANI** [comment]
Hello
When i am trying php artisan vendor:publish --tag=passport-components
it gives me this error
Unable to locate publishable resources.

I am using Laravel 8

**Comment by ERROUDANI** [comment]
> Released v9.0.1 which fixes this.

Hello
When i am trying php artisan vendor:publish --tag=passport-components
it gives me this error
Unable to locate publishable resources.

I am using Laravel 8
